core/miner/impl/miner_impl.cpp

```diff
@@ -92,40 +93,42 @@ namespace fc::miner {
     fee_config->max_precommit_batch_gas_fee.per_sector =
         TokenAmount{"2000000000000000"};
     fee_config->max_precommit_gas_fee = TokenAmount{"25000000000000000"};
+    auto addr_sel = [=](const MinerInfo &miner_info,
+                        const TokenAmount &good_funds,
+                        const std::shared_ptr<FullNodeApi> &api)
+        -> outcome::result<Address> {
+      return SelectAddress(miner_info, good_funds, api);
+    };
```

Comment:
can we remove lambda, and pass `SelectAddress` directly?

Reply:
Fixed